datatypes/Cargo.toml

```diff
@@ -11,6 +11,7 @@ edition = "2018"
 [dependencies]
 # simd compiles with nightly only, allow default features to build with arrow-flight once it is possible
 arrow = {version = "2.0", features = ["simd"] }
+bb8-postgres = { version = "0.4", features = ["with-uuid-0_8", "with-chrono-0_4", "with-serde_json-1"] }
```

Comment:
ugly dependency. can we feature gate this or limit the dependency to the ToSql/FromSql implementations?

Reply:
You mean the `postgres_types` crate?

Reply:
it seems to work without it now :shrug: 

e40a840

Reply:
516c46